tests/bsim/bluetooth/host/adv/resume2/connecter/main.c

```diff
@@ -0,0 +1,53 @@
+/* Copyright (c) 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "zephyr/bluetooth/hci_types.h"
+#include <errno.h>
+#include <testlib/conn.h>
+#include <testlib/scan.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/__assert.h>
+
+LOG_MODULE_REGISTER(connecter, LOG_LEVEL_INF);
+
+int main(void)
```

Comment:
maybe a small description of what is the job of this app (and the connectable) would be good.
I know code is doc and all that, still simpler to just read a comment at the top level

Reply:
Added